src/probnum/randvars/_normal.py

```diff
@@ -403,8 +404,11 @@ def _sub_normal(self, other: "Normal") -> "Normal":
 
     # Univariate Gaussians
     def _univariate_cov_cholesky(
-        self, damping_factor: Optional[FloatArgType] = COV_CHOLESKY_DAMPING
+        self,
+        damping_factor: Optional[FloatArgType] = None,
     ) -> np.floating:
+        if damping_factor is None:
+            damping_factor = config.covariance_inversion_damping
```

Comment:
I think it is best if in these low-level functions, the damping factor is not optional anymore (the damping factor is overwritten with the global config in "precompute_cov_cholesky" already, which is also why the test coverage is not available here :)

Reply:
Okay